include/dev/TPS2HB50BQ1.hpp

```diff
@@ -90,13 +92,12 @@ class TPS2HB50BQ1 {
     IO::GPIO& diagSelect1;
     IO::GPIO& diagSelect2;
     IO::ADC& senseOut;
+    ACS71240 currentSensor;
 
     void setDiagStateEnabled(bool state);
 
     /**
      * Read the sense out of the power switch
-     *
-     * @param senseOut The sense out value
      */
```

Comment:
This needs to describe what the output is.
```suggestion
    /**
     * Read the sense out of the power switch
     *
     * @return The raw ADC value
     */
```